client/my-sites/post-share/index.jsx

```diff
@@ -169,6 +183,135 @@ class PostShare extends Component {
 		);
 	}
 
+	renderFooterSectionItem( item, index ) {
+		const {
+			service,
+			handle,
+			timestamp,
+			message,
+		} = item;
+
+		return (
+			<CompactCard className="post-share__footer-items" key={ index }>
+				<div className="post-share__footer-item">
+					<div className="post-share__handle">
+						<SocialLogo icon={ service === 'google_plus' ? 'google-plus' : service } />
+						<span>
+							{ service === 'twitter' ? `@${ handle }` : handle }
+						</span>
+					</div>
+					<div className="post-share__timestamp">
+						<Gridicon icon="time" size={ 18 } />
+						<span>
+							{ timestamp }
+						</span>
+					</div>
+					<div className="post-share__message">
+						{ message }
+					</div>
+				</div>
+				<EllipsisMenu>
+					<PopoverMenuItem icon="visible">
+						Preview
+					</PopoverMenuItem>
+					<PopoverMenuItem icon="pencil">
+						Edit
+					</PopoverMenuItem>
+					<PopoverMenuItem icon="trash">
+						Trash
+					</PopoverMenuItem>
+				</EllipsisMenu>
+			</CompactCard>
+		);
+	}
+
+	renderScheduledList() {
+		const { planSlug, translate } = this.props;
+
+		if ( planSlug !== PLAN_BUSINESS ) {
+			return (
+				<Banner
+					className="post-share__footer-banner"
+					callToAction={ translate( 'Upgrade for $9.99' ) }
+					description={ translate( 'Live chat support and no advertising.' ) }
+					list={ [
+						translate( 'Live chat support' ),
+						translate( 'No advertising' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'No Advertising' )` *ES Score: 25*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/publicize-footer-section)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).